libs/category/src/drivers/magento/transformers/category-page-config-transformer.service.ts

```diff
@@ -1,42 +1,62 @@
 import { Injectable } from '@angular/core';
 
-import { SortFieldsAndFiltersProductNode, FilterNode } from '@daffodil/product';
+import { ProductNode } from '@daffodil/product';
 
 import { DaffCategoryPageConfigurationState } from '../../../models/category-page-configuration-state';
-import { CategoryNode } from '../models/outputs/category-node';
 import { DaffCategoryFilter, DaffCategoryFilterTypes } from '../../../models/category-filter';
+import { MagentoAggregation } from '../models/aggregation';
+import { MagentoPageInfo } from '../models/page-info';
+import { MagentoSortFields } from '../models/sort-fields';
 
 @Injectable({
   providedIn: 'root'
 })
 export class DaffMagentoCategoryPageConfigTransformerService {
 
-  transform(categoryNode: CategoryNode, sortsAndFilters: SortFieldsAndFiltersProductNode): DaffCategoryPageConfigurationState {
-    return {
-      id: categoryNode.id,
-      page_size: categoryNode.products.page_info.page_size,
-      current_page: categoryNode.products.page_info.current_page,
-      total_pages: categoryNode.products.page_info.total_pages,
-      filters: sortsAndFilters.filters.map((filter) => this.transformFilter(filter)),
-      sort_options: sortsAndFilters.sort_fields.options,
-			product_ids: null,
-			total_products: null
+  transform(
```

Comment:
This has an absurd number of arguments. Trim this down, or make it an object.

Reply:
made it an object